### ISSUE TITLE
Enhancement : PIXI.js & Live2D

### ISSUE THREAD

**Issue by KasaiDot** [DotSaikyo/Live2D#3]
Going to move the app to the new system created by @avgjs [pixi-live2d](https://github.com/avgjs/pixi-live2d), go check out his demo!
Stay stunned & be happy ;D

**Comment by KasaiDot** [comment]
It's gonna take a while to do it, but whatever.
If you want to contribute, don't hesit ;D

**Comment by KasaiDot** [comment]
Since the app is awfully documented (well, it's true that i didn't understand one thing), i'll wait to see when we can use it.

**Comment by huiyadanli** [comment]
if you use [pixi-live2d](https://github.com/avgjs/pixi-live2d), you had better rebuild your project to ECMAScript 2015, and use some tools like [webpack](http://webpack.github.io/) to manage project.

why not use [live2d-helper.js](https://github.com/huiyadanli/live2d-helper.js)?  it is very simple to use,
although it not yet has document.....  
(The difference between their: pixi-live2d use pixi.js, but live2d-helper.js not)

**Comment by KasaiDot** [comment]
@huiyadanli i'm gonna use the live2d-helper from now on, but for the v2, i'll move to the pixi-live2d who had better compatibility with other scripts ect... since it's writing in ECMAScript 2015 (though i prefer the lastest)

**Comment by Icemic** [comment]
@KasaiDot No, it's just WRITTEN in ECMA2015+, but released ES5 compatible.

You can find release version in https://github.com/avgjs/pixi-live2d/releases
